src/pointer-analysis/value_set_domain.h

```diff
@@ -116,4 +177,8 @@ void value_set_domain_templatet<VST>::transform(
   }
 }
 
+// To pass the correct location to the constructor we need a factory
+typedef ai_domain_factory_location_constructort<value_set_domaint>
+  value_set_domain_factoryt;
```

Comment:
I think we prefer `using` over `typedef` nowadays.

Reply:
Changed.